sdk/exportentities/workflow.go

```diff
@@ -12,9 +13,10 @@ import (
 
 // Workflow is the "as code" representation of a sdk.Workflow
 type Workflow struct {
-	Name        string `json:"name" yaml:"name"`
-	Description string `json:"description,omitempty" yaml:"description,omitempty"`
-	Version     string `json:"version,omitempty" yaml:"version,omitempty"`
+	Name        string  `json:"name" yaml:"name"`
+	Description string  `json:"description,omitempty" yaml:"description,omitempty"`
+	Version     string  `json:"version,omitempty" yaml:"version,omitempty"`
+	Template    *string `json:"template,omitempty" yaml:"template,omitempty"`
```

Comment:
Add a unit test with a template in exportentities package

Reply:
done